Sources/SwifterSwift/Foundation/NSAttributedStringExtensions.swift

```diff
@@ -134,6 +134,45 @@ public extension NSAttributedString {
     }
 }
 
+// MARK: - Initializers
+
+public extension NSAttributedString {
+    #if os(iOS) || os(tvOS)
```

Comment:
If you update the imports at the top to
```Swift
#if canImport(UIKit)
import UIKit

public typealias Image = UIImage
#endif

#if canImport(AppKit)
import AppKit

public typealias Image = NSImage
#endif
```
then you can unify this method into one:
```Swift
#if os(iOS) || os(tvOS) || os(macOS)
/// SwifterSwift: Creates an attributed string with an image centered by a font.
/// - Parameters:
///   - image: The image to create attributed string with (the image will not be scaled).
///   - font: The font by which the image should be centered.
///
/// https://stackoverflow.com/a/45161058
@available(macOS 10.11, *)
convenience init(image: Image, font: Font) {
    let icon = NSTextAttachment()
    icon.bounds = CGRect(x: 0, y: (font.capHeight - image.size.height).rounded() / 2,
                         width: image.size.width, height: image.size.height)
    icon.image = image

    self.init(attachment: icon)
}
#endif
```